client/src/app/core/core-services/action.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+
+import { HttpService } from './http.service';
+
+/**
+ * Currently supported actions
+ */
+export enum ActionType {
+    TOPIC_CREATE = 'topic.create',
+    TOPIC_DELETE = 'topic.delete',
+    TOPIC_UPDATE = 'topic.update',
+    AGENDA_ITEM_CREATE = 'agenda_item.create',
+    AGENDA_ITEM_UPDATE = 'agenda_item.update',
+    AGENDA_ITEM_DELETE = 'agenda_item.delete',
+    MOTION_UPDATE = 'motion.update',
+    MOTION_DELETE = 'motion.delete',
+    MOTION_SORT = 'motion.sort',
+    MOTION_UPDATE_METADATA = 'motion.update_metadata',
+    COMMITEE_CREATE = 'commitee.create',
+    MEETING_CREATE = 'meeting.create',
+    MEETING_UPDATE = 'meeting.update',
+    MEETING_DELETE = 'meeting.delete'
+}
+
+export interface RequestInfo {
```

Comment:
@GabrielInTheWorld Since the Type Variable wasn't used, I removed it. If you need it for something, feel free to add it back